tools/yaml-templates/build-test-publish.yml

```diff
@@ -106,6 +106,15 @@ steps:
       flattenFolders: true
     displayName: 'Copy validDomains for CDN'
```

Comment:
If we want it to be 7 days, should we remove this line which will still put a copy in the original 7300 days place?

Reply:
We want this for back compat with the older versions of TeamsJS

Reply:
Oh yes, makes sense. Can you add a comment above this line explaining that? Would not be crazy for a dummy like me to come along later and be like "Why do we have two of these things? DELETE"